packages/core/integration-tests/test/hmr.js

```diff
@@ -9,6 +9,7 @@ import {
   overlayFS,
   sleep,
 } from '@parcel/test-utils';
+// flowlint untyped-import:off
```

Comment:
```suggestion
// flowlint-next-line untyped-import:off
```

Reply:
The reason I didn't include the `next-line `here is that I wanted the rule applied for the next 4 lines, which all are untyped imports. the flowlint would only apply to the rest of the file, and not before so it wouldn't affect the previous imports.

Reply:
Ah, got it. Personally I would suppress each import individually, as this suppresses the violation for any imports added after this line in the future. But this is fine to me, especially if we can get typings for the remaining packages and remove it soon.

thoughts @padmaia @mischnic?

Reply:
> Personally I would suppress each import individually, as this suppresses the violation for any imports added after this line in the future.

👍

Reply:
Another option might be to group all the untyped imports and enable the lint rule afterwards with `// flowlint untyped-import:error`.